conf/modules.config

```diff
@@ -64,25 +64,65 @@
         }
 
         //
-        // Tree building
+        // Tree building (guidetree)
         //
 
-        withName: "CLUSTALO_GUIDETREE|FAMSA_GUIDETREE|MAFFT_GUIDETREE" {
+        withName: "CLUSTALO_GUIDETREE|FAMSA_GUIDETREE|MAGUS_GUIDETREE" {
```

Comment:
Mafft guidetree should be part of the sub workflow
Magus guidetree should not be part of it